map_view/display-device-location-with-autopan-modes/README.metadata.json

```
@@ -0,0 +1,37 @@
+{
+    "category": "Map",
```

we are in Map view for this sample :) is there a problem with the script?
```suggestion
    "category": "Map view",
```